sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_outbound_rule_operations.py

```diff
@@ -3,7 +3,7 @@
 # ---------------------------------------------------------
 
 from typing import Dict
-from azure.ai.ml._restclient.v2022_12_01_preview import AzureMachineLearningWorkspaces as ServiceClient122022Preview
+from azure.ai.ml._restclient.v2023_02_01_preview import AzureMachineLearningWorkspaces as ServiceClient022023Preview
```

Comment:
According to previous comment, can workspace continue to use 2022_12_01_preview version?